src/main/kotlin/hu/kirdev/schpincer/web/ConfigureController.kt

```diff
@@ -89,6 +90,40 @@ open class ConfigureController {
         return "redirect:/configure/$circleId"
     }
 
+    @GetMapping("/configure/{circleId}/members/edit/{memberId}")
+    fun editMember(@PathVariable circleId: Long,
+                   @PathVariable memberId: Long,
+                   model: Model): String? {
+        model.addAttribute("circles", circles.findAllForMenu())
+        model.addAttribute("circleId", circleId)
+        model.addAttribute("mode", "edit")
+        model.addAttribute("member", members.getOne(memberId))
+        return "memberModify"
+    }
+
+    @PostMapping("/configure/{circleId}/members/edit")
+    fun editMember(@PathVariable circleId: Long,
+                   cme: @Valid CircleMemberEntity,
+                   @RequestParam id: Long,
+                   @RequestParam avatarFile: MultipartFile?,
+                   request: HttpServletRequest
+    ): String {
+        if (cannotEditCircleNoPR(circleId, request)) return "redirect:/configure/$circleId?error"
+
+        val original = members.getOne(id)
+        if (original.circle!!.id != circleId) return  "redirect:/configure/$circleId?error";
```

Comment:
Extra `;` at end of line and extra space after `return`.

Reply:
Oops, the reflex in my finger;